backend/backend/handlers/comments/commentService.py

```diff
@@ -0,0 +1,420 @@
+#  Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  SPDX-License-Identifier: Apache-2.0
+
+import os
+import boto3
+import json
+from boto3.dynamodb.conditions import Key
+from boto3.dynamodb.types import TypeDeserializer
+from backend.common.validators import validate
+from typing import List
+
+dynamodb = boto3.resource("dynamodb")
+dynamodb_client = boto3.client("dynamodb")
+response = {
+    "statusCode": 200,
+    "body": "",
+    "headers": {
+        "Content-Type": "application/json",
+        "Access-Control-Allow-Credentials": True,
+        "Access-Control-Allow-Origin": "*",
+        "Access-Control-Allow-Headers": "Content-Type",
+        "Access-Control-Allow-Methods": "OPTIONS,POST,GET",
+    },
+}
+comment_database = None
+
+try:
+    comment_database = os.environ["COMMENT_STORAGE_TABLE_NAME"]
+except:
+    print("Failed Loading Comment Storage Environment Variables")
+    response["body"]["message"] = "Failed Loading Comment Storage Environment Variables"
+
+
+def get_all_comments(queryParams: dict, showDeleted=False) -> dict:
+    """
+    Gets all of the comments from the database
+    NOTE: This is not currently used, but could be helpful to populate the comments section when the page first loads and no asset is selected
+    :param queryParams: pagination information
+    :param showDeleted: boolean storing if deleted comments should be returned
+    :returns: all comments in the database
+    """
+    deserializer = TypeDeserializer()
+
+    paginator = dynamodb_client.get_paginator("scan")
```

Comment:
Lets get the pagination parameters (page_no, maxPerPage) from the UI. That will allow for faster load times of comments in cases where there are plenty of comments

Reply:
I am realizing I made a bit of a mistake here. 

1. This function was added for a different use case that we didn't end up implementing so the function is unused. The functions that actually get called to get the comments are get_comments() and get_comments_version() below. These functions were modeled after the functions to get assets ([backend/backend/handlers/assets/assetService.py](https://github.com/awslabs/visual-asset-management-system/blob/d14fa7a599e00b7ee4cd582bab24f1ac8eb1611f/backend/backend/handlers/assets/assetService.py#L120-L131)), and do not currently support pagination.
2. I am not using any of the Cloudscape components to display the comments because I wanted to have freedom formatting them, which means that there is no built in pagination for them like there is for the Cloudscape tables.

The solution to this would be to build a pagination method into the frontend such as scrollable pagination like Slack has, for example, or actual pages, similar to the Cloudscape tables. I would love to hear your thoughts on this.